$MirOS$
$OpenBSD: patch-libopenjpeg_opj_malloc_h,v 1.3 2012/03/30 07:09:26 sthen Exp $

Required for !amd64. We could use posix_memalign() instead, but no point
as our malloc() already aligns to 16 bytes.

--- src/lib/openjp2/opj_malloc.h.orig	Mon Nov 19 14:25:43 2012
+++ src/lib/openjp2/opj_malloc.h	Sun Aug  4 15:07:50 2013
@@ -85,8 +85,8 @@ Allocate memory aligned to a 16 byte bou
 		#define HAVE_MEMALIGN
   #elif defined(__FreeBSD__)
     #define HAVE_POSIX_MEMALIGN
-	/* Linux x86_64 and OSX always align allocations to 16 bytes */
-	#elif !defined(__amd64__) && !defined(__APPLE__) && !defined(_AIX)
+	/* Linux x86_64, OSX, AIX, OpenBSD always align allocations to 16 bytes */
+	#elif !defined(__amd64__) && !defined(__APPLE__) && !defined(_AIX) && !defined(__OpenBSD__)
 		#define HAVE_MEMALIGN
 		#include <malloc.h>			
 	#endif
